environment

```diff
@@ -23,5 +27,5 @@ SECRETSTORE_RUNNER_TOKEN="gitlab_server-runner_token"
 GITLAB_SERVER_NAME="smallspark"
 GITLAB_USERNAME="ubuntu"
 EXTERNAL_URL="biodata-integration-tests.net"
-TERRAFORM_BACKEND_STATE_BUCKET='terraform-gitlab-state-d3443c'
+TERRAFORM_BACKEND_STATE_BUCKET="terraform-gitlab-state-d3443c"
```

Comment:
What's the meaning of the suffix?

Reply:
No meaning, just makes it unique since I want to reuse the name in other accounts and google bucket names need to be globally unique.